src/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class Header extends React.Component{
+  render(){
+    return(
+      <header>
+        <h1>Gallery of Horns</h1>
+      </header>
```

Comment:
Good work here as well!

When we think of websites, we usually see the highest level with a header at the top, then a main section, then a footer at the bottom etc.

- For line 6, we have a 'header' but this entire component will serve as your header. We only need to add in the `h1`  👍